commander/src/commands/init.ts

```diff
@@ -17,7 +17,8 @@
 import BaseBootstrapCommand from '../base_bootstrap_command';
 
 export default class InitCommand extends BaseBootstrapCommand {
-	static description = 'Bootstrap a lisk-sdk project.';
+	static description =
+		'Commands relating to bootstrapping a blockchain application or plugin using lisk-sdk.';
```

Comment:
This topic description should be in the package.json?
For this specific command, it should be only showing about this command.
Also, product name is `Lisk SDK`